study-builder/studies/circadia/emails/welcome.html

```diff
@@ -0,0 +1,8 @@
+<html>
+<body>
+    <p>
+        -ddp.salutation-
```

Comment:
I recommend using dynamic templates. See our [wiki page](https://broadinstitute.atlassian.net/wiki/spaces/DDP/pages/1182630029/Emails+Guide#Dynamic-Templates) for variables you can use (e.g. `{{salutation}}`).

Reply:
This email was a blank one when we started study development. It was removed and a new one was added [here](https://github.com/broadinstitute/ddp-study-server/pull/856).